treewide: fix compatible string for ath10k

neheb opened the pull request:
The ath9k documentation says to use pci168c strings for the compatible string, probably because the OWL loader uses it to overide bogus pci IDs like abcd. This is not the case with ath10k and the documentation explicitly states to use qcom,ath10k.

ping @DragonBluep @musashino205

neheb committed:
treewide: fix compatible string for ath10k

The ath9k documentation says to use pci168c strings for the compatible
string, probably because the OWL loader uses it to overide bogus pci IDs
like abcd. This is not the case with ath10k and the documentation
explicitly states to use qcom,ath10k.

Signed-off-by: Rosen Penev <rosenp@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/18920
Signed-off-by: Robert Marko <robimarko@gmail.com>

Ansuel commented:
It's sad tho as we lose information of the actual pcie card mounted. Maybe we should update the documentation to add all these entry?

@robimarko any idea on this?

neheb commented:
I was doing led-sources conversions until I realized these pci entries are ath10k. I could add a comment stating which ath10k card it is.

DragonBluep reviewed (commented):
I've checked the wiki and git history. It can be confirmed that all devices are ath10k based, except for `compex_wpj531-16m`.

neheb commented on target/linux/ath79/dts/qca9531_compex_wpj531-16m.dts:
```diff
@@ -149,7 +149,7 @@
 
 	wifi@0,0 {
-		compatible = "pci168c,003c";
+		compatible = "qcom,ath10k";
```
There’s no nvmem stuff. Can probably just remove.

neheb replied:
Removed.

neheb committed:
ath79: compex_wpj531: remove wrong wifi device

This device does not come with a pci card. It has a slot where one can
get supplied. What's more, nvmem or ieee80211 frequency limits are not
applied here. It can just get removed.

Signed-off-by: Rosen Penev <rosenp@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/18920
Signed-off-by: Robert Marko <robimarko@gmail.com>

neheb commented:
@Ansuel rebased.

I don't think documentation should be updated. the pci168c thing is a quirk of the ath9k documentation. If it were upstreamed today it would probably not be allowed.

neheb commented:
ping?
